nova_vm/src/ecmascript/scripts_and_modules/script.rs

```diff
@@ -181,9 +181,11 @@ pub fn parse_script(
     source_text: Box<str>,
     realm: RealmIdentifier,
     host_defined: Option<HostDefined>,
+    strict_mode: bool,
```

Comment:
**nitpick:** IMO HostDefined should be last argument.

Reply:
Done.